api/managementpb/actions.proto

```diff
@@ -95,6 +95,8 @@ message StartMySQLExplainActionRequest {
   string service_id = 2;
   // query example.
   string query = 3;
+  // db which will be used for explain query.
+  string db = 4;
```

Comment:
I think we should be a bit more tidy in both public API field names and documentation from now on.
```
// The database name for EXPLAIN query.
string database = 4;
```

Reply:
Note the second line too…